src/domain/service/note.ts

```diff
@@ -0,0 +1,46 @@
+import NoteEntity from '@domain/entities/note.js';
+
+/**
+ * Interface for the add note query.
+ */
+interface AddNoteQuery {
+  /**
+   * Note title
+   */
+  title: string;
+
+  /**
+   * Note content
+   */
+  content: string;
+}
+
+/**
+ * Note service
+ */
+export default class NoteService {
+  /**
+   * Adds note
+   *
+   * @param query - add note query
+   * @returns {unknown} - added note object
+   */
+  public static addNote(query: unknown): unknown {
+    /**
+     * TODO: Validate query
```

Comment:
Validation of input data should be on the interface level

Reply:
replaced